dotcom-rendering/src/web/components/TopRightAdSlot.importable.tsx

```diff
@@ -6,6 +6,7 @@ import { ShadyPie } from './ShadyPie';
 
 const isServer = typeof window === 'undefined';
 const MOSTVIEWED_STICKY_HEIGHT = 1059;
+const LABS_HEADER_HEIGHT = 55;
```

Comment:
I originally pulled this value out of the [labs header file](https://github.com/guardian/dotcom-rendering/blob/17a61d25aa2701e3ce69da4d940d89bbab888ac5/dotcom-rendering/src/web/components/LabsHeader.importable.tsx#L21) itself. I've since gone back on this as it causes this island's bundle to pull in the entire labs header script (bundle analysis below).

![Screenshot 2022-09-12 at 16 33 40](https://user-images.githubusercontent.com/8000415/189697668-84f5d8c7-c746-44bd-b395-ba76181a4f00.png)

 Redefining the value here seems like the simplest way to avoid this.

Reply:
Not a blocker, but I would rather we had an explicit coupling if there is one.

Could you create a module that contains these constant and import these pixel values from there?

```ts
// commercialConstants.ts
const LABS_HEADER_HEIGHT = 55;

// TopRightAdSlot.importable.tsx
import { LABS_HEADER_HEIGHT } from "./commercialConstants.ts";
css`
  top: ${LABS_HEADER_HEIGHT}px;
`

// LabsHeader.importable.tsx
import { LABS_HEADER_HEIGHT } from "./commercialConstants.ts";
css`
  height: ${LABS_HEADER_HEIGHT}px;
`
```

Reply:
Good point. I'll add this here whilst this is still open.